game.libretro.dosbox/addon.xml.in

```diff
@@ -5,6 +5,7 @@
         provider-name="Libretro">
     <requires>
         <import addon="game.libretro" version="1.0.0"/>
+        <import addon="game.controller.mouse" version="1.0.0"/>
```

Comment:
Can you add game.controller.default before this line, because dosbox also supports joysticks? Will that interfere with the mouse handler?